equinix/fabric_port_mapping_helper.go

```diff
@@ -104,27 +104,6 @@ func encapsulationToTerra(portEncapsulation *v4.PortEncapsulation) *schema.Set {
 	return portEncapsulationSet
 }
 
-func lagToTerra(portLag *v4.PortLag) *schema.Set {
-	if portLag == nil {
```

Comment:
Does the API still return the fields that are needed for this function?  I raised a concern about breaking changes for the `lag` attribute elsewhere on this PR, but if the old attribute can't work anyway because the API already removed these fields, then my concern is moot.

Reply:
Yes, I believe the API is fixed now. cc @anchala71